tests/phpunit/admin/class-wc-stripe-settings-controller-test.php

```
@@ -110,4 +110,99 @@ public function test_add_buttons_action_is_called_on_order_admin_page() {
+	}
+
+	public function provide_test_admin_scripts_checkout_sessions_country_restrictions(): array {
+		return [
+			'US account + feature available'              => [ 'US', true, true ],
+			'DE account (EEA) + feature available'        => [ 'DE', true, false ],
+			'DE account (EEA) + feature unavailable'      => [ 'DE', false, false ],
+			'US account + feature unavailable'            => [ 'US', false, false ],
+			'CA account + feature available (not in EEA)' => [ 'CA', true, true ],
+		];
+	}
+}
```

PHPMD (2.15.0)

[error] The class WC_Stripe_Settings_Controller_Test is not named in CamelCase. (undefined)

(CamelCaseClassName)